src/common/utils.ts

```diff
@@ -110,3 +110,7 @@ export const safeJSONParse = <T>(json: string): T | undefined => {
     return undefined;
   }
 };
+
+export function isNumber(value: string | number): boolean {
+  return !!value && !isNaN(Number(value.toString()));
```

Comment:
this would fail if passed `0`

Reply:
we have `lodash.isnumber` which should be more bulletproof, we can add it to dependencies

Reply:
Oh, cool. Let me check that.

Reply:
Ah, yes, forgot to mention. I actually tried it but the test is breaking: https://github.com/dailydotdev/daily-api/pull/2151/commits/a79cdcf82a40fc5828300faf29aa1e859666b010

I think it is due to using `isNaN` vs `Number.isNaN`. The package is probably using the latter, but what we need is the former.

Reply:
Ok but implementation above is not correct, if we pass `isNumber(0)` it will say `false` while `0` is a number.

Reply:
It is probably related by the fact you do `!!value` which makes your `isNumber` invalid and you are missing a null check.

Reply:
@sshanzel I added some edge case covers based on https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN#examples and tests to only support number and string types, see 5fcb1255e38a525d4e3e1a0905f1fffeba3d7c35